### DISCUSSION TITLE
Adding a button for JSON to interface or class converted in the response section

### DISCUSSION THREAD

**Discussion by Samuele-Barbiera** [thunderclient/thunder-client-support#747] [Ideas]
Having the ability to add a button to copy the conversion of the JSON response of the API in an interface for typescript or a class for dart, c#, etc..., it would be great to put it beside the button for the code snippet on the top-right side

**Comment by rangav** [comment]
@Samuele-Barbiera better to create new issue instead of discussion.

thanks for the feedback

**Reply by Samuele-Barbiera** [reply]
i've created the issue
